app/src/main/res/values/strings.xml

```diff
@@ -638,5 +638,8 @@
     <string name="Search_Bar_Title">Search Experiments</string>
     <string name="Search_Bar_Hint_Text">Search..</string>
     <string name="search_menu_title">Search</string>
+    <string name="what_is_ps_lab_device">What is PSLab Device?</string>
+    <string name="please_connect_the_ps_lab_device_through_otg">Please connect the PSLab Device through OTG</string>
```

Comment:
You can try and shorten this name and also you can use pslab instead of ps_lab

Reply:
that is just Id it auto generated by android , it won't affect the real text.

Reply:
@Avjeet Auto generated IDs are simply the text for which you want to generate that id with spaces replaced by underscores and all letters in lower case. So, try to give self made IDs as these IDs are not suitable. You can rather use about_pslab_device and  connect_device IDs.